rfd/0180-sso-as-mfa-method.md

```diff
@@ -72,171 +194,197 @@ If browser window does not open automatically, open it by clicking on the link:
 
-#### Default MFA connector
+##### `mfa.mode`
 
-Cluster auth preference will also gain the `mfa_connector_name` field to set
-a preferred IdP connector for MFA checks. As detailed in the UX section, setting
-a preferred IdP connector has some beneficial UX implications.
+See [SSO MFA Modes](#sso-mfa-modes).
 
-If `mfa_connector_name` is not set, but `connector_name` is set, that connector
-will be used as an MFA method if `allow_as_mfa_method` is set to `yes`.
+##### `mfa.force_reauth` (saml)
 
-If neither of the fields above are set, Teleport will look through all
-registered connectors in lexical order and return the first one with 
-`allow_as_mfa_method` set to `yes` or`only`.
+For SAML, forced re-authentication can be achieved by adding the `ForceAuth`
+attribute to the SAML request. This field will be added to both the parent
+and MFA auth connector settings.
 
-#### Bad configuration
+##### `mfa.max_age` (github) 
 
-The following two fail states should be prevented:
-- `mfa_connector_name` points to a connector with `allow_as_mfa_method = no`
-- `connector_name` points to a connector with `allow_as_mfa_method = only`
-
-These fail states will be checked on both connector update and auth preference
-update.
+The github connector does not currently support `max_age`, so it will be
+added to both the parent and MFA auth connector settings to match the
+generic OIDC auth connector resource.
 
 ### Security
 
 Teleport uses MFA checks for some of its most security focused features, including
 per-session MFA, moderated sessions, and MFA for admin actions. Using SSO as an
-MFA method opens up the possibility of poorly configured clusters being
+MFA method opens up the possibility of poorly configured auth connectors being
 vulnerable to attacks ranging from internal users avoiding safe MFA checks to
 attackers with a compromised IdP gaining keys to the castle.
 
-#### Opt-in
+#### Strict Guidelines
 
 SSO as an MFA method will be opt-in. Administrators will be instructed through
-the docs to only enable an IdP connector as an MFA method if the IdP provider
-has strict checks itself (e.g. Administered Webauthn devices, Trusted devices).
+the docs to only enable MFA for an auth connector if the IdP provider has strict
+checks itself (e.g. Administered Webauthn devices, Trusted devices).
 
 Teleport has no way to confirm whether a registered IdP connector follows the
 guidelines, but it will display a warning to admins who attempt to enable it.
 
 ```console
 > tctl edit connector/okta
-### sets `allow_as_mfa_method: yes`
+### sets `spec.mfa.enabled=yes`
+
 Warning: Allowing this IdP provider to be used as an MFA method may reduce the
 security of enforced MFA checks for critical security features. This option
 should not be enabled unless the IdP provider has strict MFA and/or Device trust
 enforcement itself. Continue? (y/N):
 ```
 
+Additionally, forced re-authentication will be the default setting. Even if a
```

Comment:
How reliable is forced re-authentication? I remember we talked about it but it left me with the impression that it's hard to do for certain methods/providers.